packages/react/src/components/CTA/CTA.js

```diff
@@ -35,7 +35,7 @@ const CTA = ({ style, type, ...cta }) => {
           href={cta.href}
           icon={_iconSelector(type)}
           target={_external(type)}
-          handleClick={e => _jump(e, type)}
+          onMouseDown={e => _jump(e, type)}
```

Comment:
@photodow @mkothur is switching to mousedown going to have any accessibility impact? seems like it would...

Reply:
Yeah, we were just talking about that.

Reply:
he's going to revert it to handleClick again, and we'll work with Carbon on a fix.

Reply:
ok, yeah let's get this in as-is with click events, and track separately.